### PR TITLE
Removed obsolete ms.prod tag, FAQ updates, and various other misc changes

### LINKED ISSUE #125: Ubuntu 16.04 FAQ reference may be outdated
The vcpkg FAQ states that continuous testing targets Ubuntu 16.04. It is possible that this statement is out of date considering that standard support for Ubuntu 16.04 (and 18.04 for that matter) has expired.

It would be good to know what the current support target is.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9bdc4f66-0014-e830-24d0-1b462e44b7c3
* Version Independent ID: 9bdc4f66-0014-e830-24d0-1b462e44b7c3
* Content: [Frequently Asked Questions](https://learn.microsoft.com/en-us/vcpkg/about/faq)
* Content Source: [vcpkg/about/faq.md](https://github.com/Microsoft/vcpkg-docs/blob/main/vcpkg/about/faq.md)
* Product: **vcpkg**
* GitHub Login: @vicroms
* Microsoft Alias: **viromer**

### PR DESCRIPTION
**Pull request by AugP**
Removed obsolete ms.prod tag (now superseded by ms.service tag in our central docfx.json file), various formatting fixes, and updated the FAQ to address some issues.

Also fixes #125

### CODE REVIEW

**Comment by AugP** [comment]
Got a few more bugs to address on this PR before it's ready.

File: vcpkg/commands/add.md [hunk 1]
```diff
@@ -33,7 +34,7 @@ To add ports - `fmt`, `sqlitecpp`, `zlib`:
 
 ```console
 vcpkg add port fmt sqlitecpp zlib
-```
+
```
Comment (ras0219-msft) [inline vcpkg/commands/add.md:37]: Is this removal intended?

File: vcpkg/commands/create.md [hunk 2]
```diff
@@ -26,7 +26,9 @@ The command can save the downloaded source code using a specific file name, prov
 It's important to understand that the port created by the `vcpkg create` command serves merely as a starting point and, in most cases, further edits are necessary for a successful build. For more guidance on adding ports to the vcpkg curated catalog, we recommend referring to one of our [tutorials](..\examples\packaging-zipfiles.md)
 
 ## Example
+
 ```no-highlight
+
```
Comment (ras0219-msft) [inline vcpkg/commands/create.md:31]: Is this whitespace within the code block really intended?

File: vcpkg/commands/create.md [hunk 3]
```diff
@@ -71,7 +74,7 @@ The new `portfile.cmake` has the content:
 #   VCPKG_TARGET_STATIC_LIBRARY_SUFFIX
 #   VCPKG_TARGET_SHARED_LIBRARY_SUFFIX
 #
-# 	See additional helpful variables in /docs/maintainers/vcpkg_common_definitions.md
+#   See additional helpful variables in /docs/maintainers/vcpkg_common_definitions.md
```
Comment (ras0219-msft) [inline vcpkg/commands/create.md:77]: Is this whitespace change consistent with the otuput of the tool?

File: vcpkg/get_started/get-started-adding-to-registry.md [hunk 4]
```diff
@@ -58,13 +57,15 @@ Add the `vcpkg-sample-library` port that was created in the [packaging a library
 
 ```bash
 cp -R <path/to/vcpkg-sample-library> <ports/vcpkg-sample-library>
+
```
Comment (ras0219-msft) [inline vcpkg/get_started/get-started-adding-to-registry.md:60]: Did you intend to add whitespace within the code block?

**Review by ras0219-msft — APPROVED** [review]

File: vcpkg/about/faq.md [hunk 5]
```diff
@@ -1,38 +1,48 @@
...
 description: Common questions and answers about vcpkg
-ms.date: 11/30/2022
+ms.date: 01/10/2024
 ms.topic: faq
 ---
 # Frequently Asked Questions
 
+## What are classic mode and manifest mode?
+
+In general, there are two ways to manage your dependencies with vcpkg:
```
Comment (vicroms) [inline vcpkg/about/faq.md:11]:
```suggestion
There are two ways to manage your dependencies with vcpkg:
```

File: vcpkg/about/faq.md [hunk 6]
```diff
@@ -1,38 +1,48 @@
...
+
+1. **Manifest mode** lets you declare your direct dependencies, version constraints, and registries used in a file called *vcpkg.json*. This file should be included in your code repo and can be checked in to your source control system. Dependencies will be installed in a subfolder of your repo called *vcpkg_installed*. This way, each code project can have its own set of dependencies; nothing is installed system-wide. You can run vcpkg in manifest mode by running `vcpkg install` (with no other arguments), or by taking advantage of the automatic integration vcpkg manifests have for MSBuild and CMake projects. In general, we recommend using manifests for your projects over classic mode in most cases, as you have better control over your dependencies. See our [Manifest mode article](../concepts/manifest-mode.md) for more details.
```
Comment (vicroms) [inline vcpkg/about/faq.md:13]:
```suggestion
1. **Manifest mode** lets you declare your direct dependencies, version constraints, and used registries in a file called [`vcpkg.json`](../reference/vcpkg-json.md). This file should be included in your code's repository and can be checked in to your source control system. Dependencies are installed in a subfolder named `vcpkg_installed`. This way, each code project can have its own set of dependencies; nothing is installed system-wide. You can run vcpkg in manifest mode by running `vcpkg install` (with no other arguments), or by taking advantage of the automatic [integration with MSBuild](../users/buildsystems/msbuild-integration.md) and [CMake projects](../users/buildsystems/cmake-integration.md). We recommend using manifests for your projects over classic mode in most cases, as you have better control over your dependencies. See our [manifest mode article](../concepts/manifest-mode.md) for more details.
```

File: vcpkg/about/faq.md [hunk 7]
```diff
@@ -1,38 +1,48 @@
...
 
-We recommend cloning directly from [GitHub](https://github.com/microsoft/vcpkg) and using `git pull` to update the list of portfiles. Once you've updated your portfiles, `vcpkg update` will indicate any installed libraries that are now out of date.
+We recommend cloning directly from [GitHub](https://github.com/microsoft/vcpkg) and using `git pull` to update the list of portfiles.
 
 ## Can I build a private library with this tool?
 
-Yes. Follow [our packaging zlib Example](../examples/packaging-zipfiles.md) for creating a portfile using a fake URL. Then, either pre-seed the `downloads\` folder with a zip containing your private sources or replace the normal calls to `vcpkg_download_distfile` and `vcpkg_extract_source_archive` with functions that unpack your source code.
+Yes. Follow [our packaging zlib example](../examples/packaging-zipfiles.md) for creating a portfile using a fake URL. Then, either pre-seed the `downloads\` folder with a zip containing your private sources or replace the normal calls to `vcpkg_download_distfile` and `vcpkg_extract_source_archive` with functions that unpack your source code.
```
Comment (vicroms) [inline vcpkg/about/faq.md:43]:
```suggestion
Yes. Follow [our packaging example](../get_started/get-started-packaging.md) to create your own port and see the [overlay ports](../concepts/overlay-ports.md) and [registries](../concepts/registries.md) documentation to learn how to manage your private ports.
```

**Review by vicroms — CHANGES_REQUESTED** [review]
Please fix the code block with the missing delimiter to avoid issues with formatting. 
I made some suggestions for the FAQ article.

Other than that, it looks good to me.

**Review by vicroms — APPROVED** [review]

**Pull request merged**